posthog/queries/sessions/sessions.py

```diff
@@ -150,6 +150,10 @@ def _determineInterval(interval):
                 df["date"] = (df["date"].replace(day=1) + datetime.timedelta(days=32)).replace(
                     day=1
                 ) - datetime.timedelta(days=1)
+            for idx in range(len(date_range)):
```

Comment:
I fixed the skipped test! Monthly interval was missing an extra preprocessing for the dates